SignalServiceKit/src/Messages/Interactions/TSOutgoingMessage.m

```diff
@@ -541,6 +541,8 @@ - (OWSSignalServiceProtosAttachmentPointer *)buildAttachmentProtoForAttachmentId
 
     OWSSignalServiceProtosAttachmentPointerBuilder *builder = [OWSSignalServiceProtosAttachmentPointerBuilder new];
     [builder setId:attachmentStream.serverId];
+    NSString *contentType = attachmentStream.contentType;
```

Comment:
Did you want to do something with this var?

Reply:
Removed.